### DISCUSSION TITLE
Pride/2020 emote etc

### DISCUSSION THREAD

**Discussion by Lennxn** [Chatterino/chatterino2#3791] [Ideas] [answered]
For this could you have a setting that allows all of these emotes to be tabbable? that means the people who dont have them wont need to turn on the setting and the people that do have them will be able to tab the emotes

**Comment by Felanbird** [comment, marked as answer]
https://github.com/Chatterino/chatterino2/issues/3583#issuecomment-1074234607

**Reply by Lennxn** [reply]
The thing is this is just a workaround and only works if you have a specific word to send before it everytime. Maybe its the only workaround idk

**Reply by Felanbird** [reply]
> and only works if you have a specific word to send before it everytime

Did you actually read the comment, because that is not true. 
In the GIF I show the command response to be explaining that the text here does not matter, but just like any normal command, you can just type the the command response you want (the emote name in this situation) in there and it will work.
